code/src/index.css

```diff
@@ -1,13 +1,132 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  font-family: -apple-system, BlinkMacSystemFont, "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  color: #a0a0a0;
+  font-size: 14px;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  margin: 30px 40px 20px 40px;
+  font-size: 24px;
+  font-weight: bold;
+  color: #FFF;
+}
```

Comment:
According to the design-file the header should be underlined.

Reply:
Oh, right! The line was so faint I didn't even see it, maybe time to get new glasses. Thanks!